embedded-kafka/src/test/java/com/playtika/test/kafka/AbstractEmbeddedKafkaTest.java

```diff
@@ -60,15 +60,34 @@ protected void sendMessage(String topic, String message) throws Exception {
         }
     }
 
+    protected void sendTransactionalMessage(String topic, String message) throws Exception {
+        try (KafkaProducer<String, String> kafkaProducer = createTransactionalProducer()) {
+            kafkaProducer.beginTransaction();
```

Comment:
so you say, that this test case is actually failing if we don't meet required replication on broker?

Reply:
Yes.